plugins/less.js

```diff
@@ -26,25 +26,24 @@ module.exports = function (options) {
     options.ext = options.ext || 'less';
 
     return function (data, args, callback) {
-        var parser = new(lib.Parser)({
+        var options = {
             paths: args.paths,
             filename: args.context.name,
             dumpLineNumbers: 'comments'
-        });
+        };
 
         try {
```

Comment:
Is the `try...catch` really necessary? I think they don't throw synchronously anymore.